### DISCUSSION TITLE
How important are the model nuances (letter after FTXP, power)?

### DISCUSSION THREAD

**Discussion by kvoit** [revk/xESP32-Faikout#306] [General]
First of all, awesome project, thanks a lot!

I just got a quote for a Daikin FTXP 35 Comfora.  I see that the FTXPxxMxxxxx is in principle supported (possibly with a level shifter, but I would be able to make my own PCB).
I found that there are at least FTXP-M and FTXP-N models, but I fail to see the difference. Up to 3,5kW, both list support for 	BRC944B2, BRC073A1 remotes. That should mean either works somehow, right? Any idea why the the 5,0kW+ models do not support the wired remote, and if that means that they lack the port?

**Comment by revk** [comment]
Sorry,I don't, but someone else may know.

**Comment by ChielBoots** [comment]
> I just got a quote for a Daikin FTXP 35 Comfora. I see that the FTXPxxMxxxxx is in principle supported (possibly with a level shifter, but I would be able to make my own PCB). I found that there are at least FTXP-M and FTXP-N models, but I fail to see the difference. Up to 3,5kW, both list support for BRC944B2, BRC073A1 remotes. That should mean either works somehow, right? Any idea why the the 5,0kW+ models do not support the wired remote, and if that means that they lack the port?

I own an FTXP35M split unit (FTXP35M5V1B) and got my Faikin delivered and installed last weekend. It's not an exact answer to your question, but might be usefull information; I spent some time searching for it myself. 
Bot the Faikin and the S21 on the Daikin board use the same connector (JST-EH). I bought a set of this cable (5-pin version) so I recieved 5pcs. Cut off the dupont connectors and soldered them together 1:1. Works flawless, straight away, both local web interface and through MQTT in Home Assistant.

**Reply by kaymvoit** [reply]
Thanks for the feedback, but that iis exactly the combination I do not worry about.
FTXP-M is said to work in the docs, and the 3.5kW version also has support for the wired remote that the faikin is pretty much replacing.

What I concerned about is
a) Does it also work on the FTXP-N, and what is even the difference
b) Does it work on the 5kW+ models, because according to the datasheet, those do not support wired remote.

Particullarly b) might be a bit of an issue, because the faikin documentation suggests that FTXP-M works, but maybe that wasn't confirmed with 5kW+?

**Comment by txitxo0** [comment]
Interested in FTXPxxN series, up!
